packages/schematics/migrations/legacy-migrations/20180515-switch-to-nx6.ts

```diff
@@ -34,13 +34,18 @@ export default {
         execSync('npm install', { stdio: [0, 1, 2] });
       }
 
-      execSync('ng update @angular/cli --from 1.7.4 --to 6.0.1', {
-        stdio: [0, 1, 2]
-      });
-      // TODO: remove --next before releasing 6.0.0 final
-      execSync('ng update @nrwl/schematics --from 1.0.3 --to 6.0.0 --next', {
-        stdio: [0, 1, 2]
-      });
+      execSync(
+        'ng update @angular/cli --from 1.7.4 --to 6.0.1 --migrate-only',
```

Comment:
`--migrate-only` skips the installation phase which is a bit annoying..
It's needed to enforce that migrations should run even though it sees the latest version in the `package.json`?